lambda_src/lambda_function.py

```diff
@@ -128,10 +132,10 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
             row_result = process_row(*path, **process_row_params)
             LOG.debug(f'Got row_result for URL: {process_row_params.get("url")}.')
 
-            if write_uri:
+            if parameterized_write_uri:
                 # Write s3 data and return confirmation
                 row_result = destination_driver.write(  # type: ignore
-                    write_uri, batch_id, row_result, row_number
+                    parameterized_write_uri, batch_id, row_result, row_number
```

Comment:
seems it would be easier to just pass `format(write_uri)` here instead of creating a separate variable for it?

Reply:
ty, a comment might do well to clarify, altho it does seem relatively straightforward now that it's just a format. there's a couple of reasons I've seen extra variables used, but not seeing them here.